tsc/meetings/2024-11-05.md

```diff
@@ -0,0 +1,98 @@
+Minutes from OpenVDB TSC meeting, October 29th, 2024
+
+Attendees: *Ken* M., *Andre* P, *Dan* B., *Greg* H, *Nick* A.
+
+Additional Attendees: Jonathan Swartz (NVIDIA), Barry Dempsey
+
+Regrets: *Jeff* L., *Richard* J.
+
+Agenda:
+
+1) Confirm quorum
+2) Secretary
+3) Meeting Times
+4) V12 post mortum
+5) 12.1 release
+6) File format updates
+7) Dilation
```

Comment:
8) Next meeting. Please correct me if I'm wrong: I think we agreed that the next meeting is Wednesday November 20, 2024 at the same time.